### ISSUE TITLE
Expression("1.1.000000000000001^1234567").eval() takes a very long time 

### ISSUE THREAD

**Issue by 835127729** [RobluScouting/EvalEx#8]
as the title metioned, code
`Expression("1.000000000000001^1234567").eval();`
takes a very long time to finish caculating.

**Comment by widavies** [comment]
Thanks, I'll fix this when I get a chance.

**Comment by HARUM1122** [comment]
I was facing the same issue therefore, i simply used math.pow. It will give you the result instantly.

**Comment by widavies** [comment]
Yeah, this library is supposed to support arbitrary precision (> 64 bit math) but my power implementation is really slow. I'll work on an optimized one when I have a chance. Thanks.
